components/title/title.module.scss

```diff
@@ -0,0 +1,5 @@
+@import 'variables.scss';
+.title{
+  color: $color-text-95;
+  font-size: $font-size-huge;
+}
```

Comment:
Create the class emoji to add those 24px of margin left to separate the text and the emoji.

Reply:
Done!